app/Http/Controllers/DocumentsController.php

```diff
@@ -82,4 +81,17 @@ public function import(Request $request)
             }
         }
     }
-}
+   public function destroy($id)
+	{
+	$document=Document::find($id);
+	$settings = Setting::findOrFail(1);
+	$companyname = $settings->company;
+	$path = $document->path;
+	$destroy_path = (public_path() . '/files/' . $companyname .'/' . $path);
+	File::delete(public_path() . '/files/' . $companyname .'/' . $path);
+	$document->delete();
+	Session()->flash('flash_message', 'Файл успешно удален!');
```

Comment:
Translate to English, please consider using Laravels translation, as that is use throughout the application